api/commands/build.ts

```diff
@@ -134,6 +134,7 @@ export class Builder {
       name: assertion.name,
```

Comment:
you can get rid of a bunch of this if you use `...assertion` and just fill the `type` and `tasks` fields

Reply:
As far as I can tell, the create proto method doesn't remove extra fields, so this actually makes me a bit uncomfortable.

Reply:
hmm, OK. perhaps academic, but i would argue that if some code is looking up *all* properties on some object (which may happen to be a protobuf), it's that code's responsibility to get it right (e.g. if looping through properties on some protobuf object, should really only fetch properties that exist in the .proto file, i.e. those in `keyOf I<proto_name>`).

(IMO this is equivalent to passing around an implementation of an interface in Java; if some code is doing something sketchy to get access to members of that object that aren't defined on the interface, that code has the responsibility to get it right)